tests/qsfs_check_metrics/qsfs_test.go

```diff
@@ -42,25 +39,23 @@ func TestMultiNodeDeployment(t *testing.T) {
 	// Check that the outputs not empty
 	metrics := terraform.Output(t, terraformOptions, "metrics")
 	assert.NotEmpty(t, metrics)
+	metrics, err := tests.IPFromCidr(metrics)
+	assert.NoError(t, err, "error parsing metrics IP")
 
-	ygg_ip := terraform.Output(t, terraformOptions, "ygg_ip")
-	assert.NotEmpty(t, ygg_ip)
-
-	status := false
-	status = tests.Wait(ygg_ip, "22")
-	if status == false {
-		t.Errorf("public ip not reachable")
-	}
-
-	verifyIPs := []string{ygg_ip, metrics}
-	tests.VerifyIPs("", verifyIPs)
+	yggIP := terraform.Output(t, terraformOptions, "ygg_ip")
+	assert.NotEmpty(t, yggIP)
+	yggIP, err = tests.IPFromCidr(yggIP)
```

Comment:
terraform outputs ygg_ip like `300:9c4a:29ac:6e0b:66b:7af1:f067:b119` so it doesn't need parsing i think